sphinx/source/_templates/docs-navbar.html

```diff
@@ -0,0 +1,36 @@
+<div class="container-xl">
+
+    <a class="navbar-brand" href="{{ pathto(master_doc) }}">
+      <img src="https://static.bokeh.org/branding/logos/bokeh-logo.svg" class="logo" alt="The Bokeh logo">
+    </a>
+
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbar-menu" aria-controls="navbar-menu" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+    </button>
+
+    {% set navbar_class, navbar_align = navbar_align_class() %}
+    <div id="navbar-menu" class="{{ navbar_class }} collapse navbar-collapse">
+      <ul id="navbar-main-elements" class="navbar-nav {{ navbar_align }}">
+        <li class="nav-item dropdown">
+        <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+          {{ VERSION }}
+        </a>
+        <div class="dropdown-menu" aria-labelledby="navbarDropdown" id="version-menu"></div>
+        </li>
```

Comment:
Is it correct that only those selected lines above are added compared to the base theme template? 

I think that https://github.com/pydata/pydata-sphinx-theme/pull/355 should be able to help make this adaptation a bit cleaner.